Application/SolutionScripts/InitializeDevelopmentEnvironment.psm1

```diff
@@ -427,19 +422,117 @@ function Invoke-PesterTests {
 
 function Invoke-PostmanIntegrationTests {
     Invoke-Task -name $MyInvocation.MyCommand.Name -task {
-        & (Get-RepositoryResolvedPath "logistics\scripts\Invoke-PostmanIntegrationTests.ps1")
+        & (Get-RepositoryResolvedPath "logistics/scripts/Invoke-PostmanIntegrationTests.ps1")
     }
 }
 
 function Invoke-SmokeTests {
     Invoke-Task -name $MyInvocation.MyCommand.Name -task {
-        & (Get-RepositoryResolvedPath "logistics\scripts\run-smoke-tests.ps1")
+        & (Get-RepositoryResolvedPath "logistics/scripts/run-smoke-tests.ps1")
     }
 }
 
 function Invoke-DotnetTest {
     Invoke-Task -name $MyInvocation.MyCommand.Name -task {
-        & (Get-RepositoryResolvedPath "logistics\scripts\run-tests.ps1")
+        & (Get-RepositoryResolvedPath "logistics/scripts/run-tests.ps1")
+    }
+}
+
+function Get-DefaultNuGetProperties {
+    $buildConfiguration = 'debug'
+    if (-not [string]::IsNullOrWhiteSpace($env:msbuild_buildConfiguration)) { $buildConfiguration = $env:msbuild_buildConfiguration }
+
+    return @(
+        "authors=Ed-Fi Alliance"
+        "owners=Ed-Fi Alliance"
+        "configuration=$buildConfiguration"
+    )
+}
+
+function New-DatabasesPackage {
+    param(
+        [string] $ProjectPath,
+
+        [string] $PackageDefinitionFile = "$ProjectPath/$PackageId.nuspec",
+
+        [string] $PackageId,
+
+        [string] $Version,
+
+        [string[]] $Properties = @(),
+
+        [string] $OutputDirectory
+    )
+
+    Invoke-Task -name "$($MyInvocation.MyCommand.Name) ($(Split-Path $ProjectPath -Leaf))" -task {
+
+        Write-Host -ForegroundColor Magenta "& `"$ProjectPath/prep-package.ps1`" $PackageId"
+        & "$ProjectPath/prep-package.ps1" $PackageId
+        Write-Host
+
+        $nuget = Install-NuGetCli -ToolsPath $ToolsPath
+
+        $params = @{
+            PackageDefinitionFile = (Get-ChildItem "$ProjectPath/$PackageId.nuspec")
+            PackageId             = $PackageId
+            Version               = $Version
+            Properties            = $Properties
+            OutputDirectory       = $OutputDirectory
+            NuGet                 = $nuget
+        }
+        New-Package @params | Out-Host
+    }
+}
+
+function New-WebPackage {
+    param(
+        [string] $ProjectPath,
+
+        [string] $PackageDefinitionFile = "$ProjectPath/bin/**/**/publish/$(Split-Path $ProjectPath -Leaf).nuspec",
```

Comment:
Double ** here?  Does it not work as a 0 to many folder expansion for powershell?

Reply:
Nope, there is just one too many wildcards.